src/cartridges/int_adyen_overlay/cartridge/scripts/adyenZeroAuth.js

```diff
@@ -25,6 +25,7 @@ const URLUtils = require('dw/web/URLUtils');
 /* Script Modules */
 const AdyenHelper = require('*/cartridge/scripts/util/adyenHelper');
 const adyenCheckout = require('*/cartridge/scripts/adyenCheckout');
+const Logger = require('dw/system/Logger');
```

Comment:
Lingering Logger declaration

Reply:
this one is actually needed because of this https://github.com/Adyen/adyen-salesforce-commerce-cloud/blob/develop/src/cartridges/int_adyen_overlay/cartridge/scripts/adyenZeroAuth.js#L58